test/benchmark/benchmarking_support.rb

```diff
@@ -36,7 +36,7 @@ def ams(label = nil, time:, disable_gc: true, warmup: 3, &block)
         version: ::ActiveModel::Serializer::VERSION.to_s,
         rails_version: ::Rails.version.to_s,
         iterations_per_second: entry.ips,
-        iterations_per_second_standard_deviation: entry.stddev_percentage,
+        iterations_per_second_standard_deviation: entry.error_percentage,
```

Comment:
@NullVoxPopuli can you update the gemfile to ensure we use the correct version of benchmark-ips?